src/main/java/org/springframework/samples/petclinic/service/ClinicServiceImpl.java

```diff
@@ -49,248 +39,248 @@
 @Service
 public class ClinicServiceImpl implements ClinicService {
 
-    private PetRepository petRepository;
-    private VetRepository vetRepository;
-    private OwnerRepository ownerRepository;
-    private VisitRepository visitRepository;
-    private SpecialtyRepository specialtyRepository;
-	private PetTypeRepository petTypeRepository;
+    private final PetRepository petRepository;
+    private final VetRepository vetRepository;
+    private final OwnerRepository ownerRepository;
+    private final VisitRepository visitRepository;
+    private final SpecialtyRepository specialtyRepository;
+    private final PetTypeRepository petTypeRepository;
 
     @Autowired
```

Comment:
nitpick: by the way, we could remove this uncessary `@Autowired`

Reply:
I agree that this annotation is optional, but for readability, as well as to make it easier for a beginner to figure out how it is injected, it seems to me better to leave this annotation as it is.

Reply:
I don't agree, but that's debatable.
For your information, the canonical version of Spring Petclinic didn't use the `@Autowired`
https://github.com/spring-projects/spring-petclinic/blob/main/src/main/java/org/springframework/samples/petclinic/owner/OwnerController.java